reactive/kotlinx-coroutines-reactive/src/Migration.kt

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2016-2019 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+@file:JvmMultifileClass
+@file:JvmName("FlowKt")
+
+package kotlinx.coroutines.reactive
+
+import kotlinx.coroutines.*
+import kotlinx.coroutines.flow.*
+import org.reactivestreams.*
+
+// Binary compatibility with Spring 5.2 RC
+@Deprecated(
+    message = "Replaced in favor of ReactiveFlow extension, please import kotlinx.coroutines.reactive.* instead of kotlinx.coroutines.reactive.FlowKt",
+    level = DeprecationLevel.ERROR
```

Comment:
It should have `DeprecationLevel.HIDDEN` since it is for binary compatibility one. Same for all other "Migration" function with the exception of `asFlow(batchSize)` which is also needed for source migration.